.github/DISCUSSION_TEMPLATE/css-battles.yml

```
@@ -0,0 +1,48 @@
+title: "CSS Battle #NUMBER – NAME"
+labels: ["CSS Battles"]
+body:
+  - type: input
+    id: link
+    attributes:
+      label: "Link to battle:"
+      description: Replace NUMBER with current battle number      
+      value: |
+       [Lets battle! ⚔️](https://cssbattle.dev/play/NUMBER)
+    validations:
+      required: true
+  - type: markdown
+    attributes: 
+      value: |
+        **That’s all! You can now click the “Start discussion” button.** :smile:
+        **Please note:** Don't change text areas - they need to stay that way to be visible after creating discussion.

```

Let's update this for a little nicer formatting and to call it out a bit more:

```suggestion
        > **Please note**
        > Don't change text areas - they need to stay that way to be visible after creating discussion
        > 
        > Don't change anything below this line!
        
        ---

```